capture/project_library/ot_trace_library/trace_library.py

```diff
@@ -47,11 +47,16 @@ class TraceLibrary:
     """
     def __init__(self, db_name, trace_threshold, wave_datatype = np.uint16,
                  overwrite = False):
+        # If .db extension is not provided, add it to the file.
+        if ".db" not in db_name:
```

Comment:
```suggestion
        if not db_name.endswith(".db"):
```
I think this would be more explicit and prevent obscure problems for which a `.db` suffix would get added unexpectedly (e.g., when a user provides `my.dbase` as `db_name`)

Reply:
Ah yes, thanks for suggestion!